crates/api_models/src/health_check.rs

```diff
@@ -0,0 +1,19 @@
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct RouterHealthCheckResponse {
+    pub database: String,
+    pub redis: String,
+    pub locker: LockerHealthResponse,
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub struct LockerHealthResponse {
+    pub status: String,
+    pub key_custodian_status: KeyCustodianStatus,
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub enum KeyCustodianStatus {
+    Unavailable,
+    Locked,
+    Unlocked,
+}
```

Comment:
No need for this, the locker health check provides context around this. We only need to know if locker is available or not